app/content/serializers/registration.py

```diff
@@ -35,6 +41,22 @@ def get_survey_submission(self, obj):
     def get_has_unanswered_evaluation(self, obj):
         return obj.user.has_unanswered_evaluations_for(obj.event)
 
+    def get_order(self, obj):
+        orders = obj.event.orders.filter(user=obj.user)
+        if len(orders):
+            # TODO write test for this that 0 is the lates order.
+            return OrderSerializer(orders[0]).data
```

Comment:
```suggestion
        order = obj.event.orders.filter(user=obj.user).first()
        if order:
            return OrderSerializer(order).data
```
Om kommentaren da er ferdig kan man skrive dette en del cleanere

Reply:
Endret nå.